content/en/docs/concepts/architecture/cloud-controller.md

```diff
@@ -107,7 +105,9 @@ routes appropriately. It requires Get access to Node objects.
 
 ### Service controller {#authorization-service-controller}
 
-The service controller listens to Service object Create, Update and Delete events and then configures Endpoints for those Services appropriately (for EndpointSlices, the kube-controller-manager manages these on demand).
+The service controller listens to Service object Create, Update and Delete events and then
```

Comment:
Optional further cleanup:
```suggestion
The service controller watches for Service object **create**, **update** and **delete** requests,
and then
```

and an equivalent change everywhere else.

Within the Kubernetes API (eg RBAC types), these verbs are written all-lowercase. See https://kubernetes.io/docs/reference/using-api/api-concepts/ for other precedents.